scripts/wasmd/generate_template.sh

```diff
@@ -47,7 +47,7 @@ function inline_jq() {
   inline_jq "template/.wasmd/config/genesis.json" -S
 
   # Custom settings in config.toml
-  sed -i "" \
+  sed -i"" \
```

Comment:
This unfortunately will not work in the mac. Can you use

```
gnused="$(command -v gsed || echo sed)"
```

and

```
"$gnused" -i"" \
```

Reply:
You can change it back as well. I remembered this was a trick that once worked on both.

Since I rarely use this repo, you can make it OSX easier.

Reply:
Reverted.